cpp/src/io/json/json_quote_normalization.cu

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023-2024, NVIDIA CORPORATION.
```

Comment:
why did the copyrights drop the year 2023?

Reply:
This is a new file in this PR implementing the normalization FST, so I think only year 2024 is included in the copyright.